cask2formula

```diff
@@ -83,11 +83,12 @@ class CaskTransform < Parslet::Transform
         commands1 = before.select{|c| c != ""}
         camelcase = name.to_s.gsub('"', "").gsub(/(^|-)(\w)/) { $2.upcase }
         commands2 = after.select{|c| c != ""}+["test do\n  end"]
+        parent = "parent = %x( [ ${PWD%/*} != ${HOMEBREW_TEMP=-/tmp} ] && echo -n '../' )\n    "
         <<~EOS
         class #{camelcase} < Formula
           #{commands1.join("\n  ")}
           def install
-            #{fonts.join("\n    ")}
+            #{parent if fonts.to_s.include? "\#{parent}"}#{fonts.join("\n    ")}
```

Comment:
According to the above changes, `fonts.to_s.include? "\#{parent}"` is always true, isn't it?

Reply:
Not always. If font formula doesn't have any directory, false. I mean font regex adds #{parent} instead of ../ so with that I don't use unnecessary check for every font. If there is parent variable it is true. Maybe I couldn't explain well. Check different examples like font-fira-code and font-cascadia
I'm not sure I fully understand your question. Is this the answer to your question?

Reply:
Oh, I see. Thanks for that you answered fully. 😉